README.md

```diff
@@ -30,7 +30,7 @@ These are configurations on which lowRISC is focusing for performance evaluation
 
 Notes:
 
-* Performance numbers are based on Cormark running on the Ibex Simple System [platform](examples/simple_system/README.md).
+* Performance numbers are based on Coremark running on the Ibex Simple System [platform](examples/simple_system/README.md).
```

Comment:
```suggestion
* Performance numbers are based on CoreMark running on the Ibex Simple System [platform](examples/simple_system/README.md).
```

Reply:
Thanks @imphil . There have been several occurences of Coremark/coremark in the documentation. I now changed it to CoreMark wherever possible. I will factor out this commit into a separate PR tough.

Reply:
It's now here #1015 .